### PR TITLE
Added title, paragraph & link to home page

### PR DESCRIPTION
**Pull request by stephlaf**
@Carms-dev Please review my `code`
Screen Shot 2021-02-22 at 10 31 53

### CODE REVIEW

**Comment by Carms-dev** [comment]
Amazingg progress!!

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,7 +1,12 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<h2>Welcome to our web app!</h2>
```
Comment (Carms-dev) [inline app/views/pages/home.html.erb:1]: For SEO, suggest to update from h2 to h1

Comment (stephlaf) [inline app/views/pages/home.html.erb:1]: Should be an h1 for SEO purposes

Comment (stephlaf) [reply app/views/pages/home.html.erb:1]: Oh, right. Fixing right away!

**Pull request merged**